docs/topics/db/models.txt

```diff
@@ -720,6 +720,16 @@ Django places some restrictions on model field names:
 
 #. A field name cannot end with an underscore, for similar reasons.
 
+#. Additionally, the word check is a reserved keyword in Django and cannot be
+   used as a field name. This is because check is a special method in Django models.
+   To work around this limitation, consider using another field name, or you can use
+   db_column as described below. For example::
+
+
+       class Example(models.Model):
+           field_check = models.IntegerField(db_column="check")
+           # Use 'db_column' to retain 'check' in the database.
```

Comment:
To follow the structure of other bullets, I think we should have something like:
```suggestion
#. A field name cannot be 'check', as this would override the check framework's
   Model.check() method.
```

Reply:
Thank you for your suggestion! I've incorporated your changes into the pull request.